screen/wallets/transactions.js

```diff
@@ -46,8 +48,8 @@ const buttonFontSize =
 
 const WalletTransactions = () => {
   const { wallets, saveToDisk, setSelectedWallet, walletTransactionUpdateStatus, isElectrumDisabled } = useContext(BlueStorageContext);
+  const walletID = wallets?.length > 0 ? wallets[0].getID() : undefined;
```

Comment:
Could be simplified like `const walletId = wallets?.[0]`?

Reply:
No, what if `wallets` is defined but empty? Then accessing index 0 should throw an error, right?

Reply:
No, it's just undefined 😁

Reply:
okay still learning... on iOS & Swift this would blow up. As an array could be initialized, but empty and you would try to access and index, which is out of bounds

Reply:
I think in Javascript arrays are also just objects 😆